### PR TITLE
add libqrencode/4.1.1

### PR DESCRIPTION
**Pull request by theirix**
Specify library name and version:  **libqrencode/4.1.1**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libqrencode/all/patches/0003-cmake-fix-install-4.1.1.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 773e037..9ffd002 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -133,7 +133,7 @@ configure_file(libqrencode.pc.in libqrencode.pc @ONLY)
+ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/qrencode.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
+ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libqrencode.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+ install(FILES qrencode.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+-install(TARGETS qrencode DESTINATION ${CMAKE_INSTALL_LIBDIR})
++install(TARGETS qrencode ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```
Comment (madebr) [inline recipes/libqrencode/all/patches/0003-cmake-fix-install-4.1.1.patch:10]: We should upstream this patch.
I apparently forgot to do this when I created the original recipe.

**Review by madebr — APPROVED** [review]

File: recipes/libqrencode/all/patches/0003-cmake-fix-install-4.1.1.patch (hunk 1 above)
Comment (theirix) [reply recipes/libqrencode/all/patches/0003-cmake-fix-install-4.1.1.patch:10]: Agree

**Review by uilianries — DISMISSED** [review]

**Comment by madebr** [comment]
Why are there 10 more packages built with the latest release?
@uilianries I think your latest open/close did only partially fix the problem.

**Comment by uilianries** [comment]
@madebr Probably because some build already passed. Same case as on GDAL: https://github.com/conan-io/conan-center-index/pull/2913#issuecomment-704362482

**Comment by uilianries** [comment]
@madebr An empty commit should rebuild all again.

**Comment by madebr** [comment]
@theirix 
We need a placebo commit!

**Comment by theirix** [comment]
@uilianries done!

**Review by madebr — APPROVED** [review]

File: recipes/libqrencode/all/patches/0003-cmake-fix-install-4.1.1.patch (hunk 1 above)
Comment (prince-chrismc) [reply recipes/libqrencode/all/patches/0003-cmake-fix-install-4.1.1.patch:10]: Maybe also add a section https://github.com/fukuchi/libqrencode#building-libqrencode-with-vcpkg for Conan =p

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**